### PR TITLE
frontend: created CoinInput for send screen

### PR DESCRIPTION
**Pull request by shonsirsha**
as a part of the refactoring effort of `send.tsx`, we're creating a new component CoinInput as its own component to be used in the send screen.

### CODE REVIEW

**Review by thisconnect — APPROVED** [review]
untested LGTM with suggestion

File: frontends/web/src/routes/account/send/send.tsx [hunk 1]
```diff
@@ -358,27 +359,6 @@ class Send extends Component<Props, State> {
...
-      this.convertToFiat(value);
-    }
-    this.setState(prevState => ({
-      ...prevState,
-      [target.id]: value,
-    }), () => {
-      this.validateAndDisplayFee(true);
-    });
-  };
-
```
Comment (shonsirsha) [inline frontends/web/src/routes/account/send/send.tsx:381]: @thisconnect In regards to your comment, the whole function can actually be removed totally. This was previously used in `sendToSelf`, but it serves no purpose as the `value` when it gets called from `sendToSelf` will be undefined (it's a label).

File: frontends/web/src/routes/account/send/send.tsx [hunk 2]
```diff
@@ -418,9 +398,10 @@ class Send extends Component<Props, State> {
     }
   };
 
-  private sendToSelf = (event: React.SyntheticEvent, recipientAddress: string) => {
-    this.setState({ recipientAddress });
-    this.handleFormChange(event);
+  private sendToSelf = (recipientAddress: string) => {
+    this.setState({ recipientAddress }, () => {
+      this.validateAndDisplayFee(true);
+    });
```
Comment (shonsirsha) [inline frontends/web/src/routes/account/send/send.tsx:404]: `sendToSelf` only needs to set `recipientAddress` to be our own address like this. We previously called `handleFormChange` but I believe there's no point in doing so.

**Comment by shonsirsha** [comment]
Rebased and made a small change to remove unused fn, PTAL 🙏

File: frontends/web/src/routes/account/send/send.tsx (hunk 2 above)
Comment (thisconnect) [reply frontends/web/src/routes/account/send/send.tsx:403]: very nice, 👍 

actually, the sendToSelf function is now the same as onReceiverAddressInputChange, I think you can remove sendToSelf and use onReceiverAddressInputChange.

**Review by thisconnect — CHANGES_REQUESTED** [review]

**Comment by shonsirsha** [comment]
Thanks! good catch. done! PTAL @thisconnect

**Review by thisconnect — APPROVED** [review]
LGTM :thumbsup:

**Pull request merged**